src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
My only real hesitation with your methods you made (`dontRender` and `doRender`) is the naming. If we could find something I liked better, I'm fine with keeping them. How about `Graph.prototype.enableAutoRender()` and `Graph.prototype.disableAutoRender()`?